docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+
+
+# Managing neuroscience projects with **datashuttle**
+*Create, validate and transfer standardised project folders*
+
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Maintaining a well-organised neuroscience project is hard. 
+
+Although everyone can appreciate the benefits of a tidy project
+folder, the practicalities of running an experiment often gets 
+in the way. Folder organisation 
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
+The best protection against such errors is automating the process
```

Comment:
This sentence is a bit long-winded, because of the dashes etc. Consider rewording or breaking up.

Reply:
How about:

```
The best protection against such errors is automating the process
through acquisition scripts—resulting in hours spent writing 
data-management code entirely unrelated to the central research goals.
```
I am also not sure whether so say `your` research goals rather than `the` to make it more personal. But it might be too much.

Reply:
I'd use "your" in this case, but more consistency throughout is more important.